src/mmw/js/src/modeling/views.js

```diff
@@ -225,6 +229,10 @@ var ScenarioTabPanelView = Marionette.ItemView.extend({
         });
         share.render();
         share.$el.modal('show');
+    },
+
+    duplicateScenario: function() {
+        this.model.collection.duplicateScenario(this.model.cid);
```

Comment:
Never realized we could access a model's collection like this! Will definitely simplify code I write in the future.